### PR TITLE
Packages: add Truchas

### PR DESCRIPTION
**Pull request by certik**

### CODE REVIEW

**Comment by certik** [comment]
@nncarlson can you please review this?

File: _data/package_index.yml [hunk 1]
```diff
@@ -638,7 +638,15 @@
...
   tags: fluid dynamics fluid simulation computational fluid dynamics turbulence high performance computing hpc cfd
   version: none
-  
+
+- name: Truchas
+  github: truchas/truchas-release
+  description: 3D Multiphysics Simulation of Metal Casting and Processing
+  categories: scientific
+  tags: fluid dynamics metal casting multiphysics hpc
+  license: BSD
```
Comment (LKedward) [inline _data/package_index.yml:647]:
```suggestion
  license: BSD 3-Clause
```

**Comment by nncarlson** [comment]
Good to me, though I'm not sure about the "version 3.0" bit.  That's the the last official tarball we rolled, but it's very old and no one should use it.  Everyone works off the master branch version.  I've done a bad job of managing versioning (help @certik :-).

**Comment by certik** [comment]
Ok, I removed the version. I am going to merge it now. Thanks @nncarlson for reviewing. Yes, we need to fix the Truchas's release process.

**Pull request merged**